masakari-controller/controller/masakari_util.py

```diff
@@ -696,6 +698,31 @@ def disable_host_status(self, hostname):
             self.rc_util.syslogout(error_code + msg, syslog.LOG_ERR)
             raise
 
+    def force_host_down_pf9(self, hostname):
+        """
+        Force host service status as "down"
+        :param hostname:
+        :return:
+        """
+        try:
+            self.rc_util.syslogout('Force nova-compute down on %s' % hostname,
+                                   syslog.LOG_INFO)
+            token = self.auth_session.get_token()
```

Comment:
The `get_token()` method is marked to be deprecated in favor of using `get_auth_headers()` which returns a dictionary. If you need the token then you could use `get_auth_headers()['X-Auth-Token']`